examples/example-gateway/config/test.json

```diff
@@ -45,6 +45,7 @@
 	"clients.withexceptions.timeout": 10000,
 	"clients.withexceptions.maxConcurrentRequests": 1000,
 	"clients.withexceptions.errorPercentThreshold": 20,
+	"contextlogger.zanzibar.skiplogs": false,
```

Comment:
I think you should add this config in test.yaml and production.yaml with default value in zanzibar.

Reply:
done